main.py

```diff
@@ -907,6 +881,7 @@ def load_all_evcs(self):
         for circuit_id, circuit in circuits:
             if circuit_id not in self.circuits:
                 self._load_evc(circuit)
+        emit_event(self.controller, "evcs_loaded", content=dict(circuits))
```

Comment:
Once you merge `master` into your branch or the targeted branch this diff isn't expected to show up, heads up.

Reply:
This line added by my commit in this PR, somehow @.@

Reply:
Looks like `dev.in` has a conflict to so going to merge branches when `kytos` and `topology` are ready to land.

Reply:
Merged master